tiledb/CMakeLists.txt

```diff
@@ -745,12 +747,12 @@ if (WIN32)
   if (TILEDB_S3)
     # the AWSSDK does not include links to some transitive dependencies
     # ref: github<dot>com/aws<slash>aws-sdk-cpp/issues/1074#issuecomment-466252911
-    target_link_libraries(TILEDB_CORE_OBJECTS_ILIB INTERFACE userenv ws2_32 wininet winhttp bcrypt version secur32)
+    target_link_libraries(TILEDB_CORE_OBJECTS_ILIB INTERFACE userenv ws2_32 wininet winhttp bcrypt ncrypt crypt32 version secur32)
   endif()
 
   if (TILEDB_AZURE)
-    # WebServices and crypt32 needed by Azure storage on windows
-    target_link_libraries(TILEDB_CORE_OBJECTS_ILIB INTERFACE WebServices crypt32)
+    # WebServices, winhttp and crypt32 needed by Azure storage on windows
```

Comment:
Should we open an issue upstream to request adding this to the cmake linkage interface?

Reply:
[They do add it.](https://github.com/Azure/azure-sdk-for-cpp/blob/9d97c40d3fae3a0f9ecf71ff97edf7a653a5b389/sdk/core/azure-core/CMakeLists.txt#L185) It is because of our arrangement with installing the static library that we have to explicitly specify our transitive dependencies as well. Building `tiledb_shared` has always been successful.